.github/workflows/static.yml

```diff
@@ -0,0 +1,29 @@
+name: Check collectstatic
+on: [push, pull_request]
```

Comment:
Allows triggering a build from the UI:
```suggestion
on: [push, pull_request, workflow_dispatch]
```

Reply:
Is this necessary? Its mainly just a check to ensure that a code change doesn't break our collect static step.